package.json

```diff
@@ -64,6 +64,7 @@
     "postcss-loader": "^2.0.10",
     "postcss-prepend-selector": "^0.3.1",
     "react-dev-utils": "^4.0.0",
+    "react-intl-gettext": "^5.1.1",
     "react-test-renderer": "^16.1.0",
```

Comment:
Not related to this, but this still needs a version bump, yeah?

Reply:
does "this" refer to package.json in general? We do need to add that still, I was going to coordinate with @thallada since I think https://github.com/edx/studio-frontend/pull/112 is landing shortly too